imod/mf6/riv.py

```diff
@@ -184,6 +186,21 @@ def _validate(self, schemata, **kwargs):
 
         return errors
 
+    def cleanup(
+        self, dis: Union[StructuredDiscretization, VerticesDiscretization]
+    ) -> None:
+        """
+        Clean up package inplace. This method calls
+        :func:`imod.prepare.cleanup.cleanup_riv`, see documentation of that
+        function for details on cleanup.
+
+        dis: imod.mf6.StructuredDiscretization | imod.mf6.VerticesDiscretization
+            Model discretization package.
+        """
+        dis_dict = {var: dis.dataset[var] for var in ["idomain", "bottom"]}
```

Comment:
I would skip the for loop and directly declare the dictionary
```
dis_dict = {
  "idomain": dis.dataset["idomain"], 
  "bottom":  dis.dataset["bottom"]
}
```

Reply:
Removed the dict comprehension